package.json

```diff
@@ -45,7 +45,7 @@
     "express": "^4.13.4",
     "geojson-validation": "^0.1.6",
     "mapbox-gl": "^0.19.0",
-    "mapbox-gl-js-mock": "^0.20.0",
+    "mapbox-gl-js-mock": "^0.21.0",
```

Comment:
I added some missing methods and incremented the version.

mapbox-gl-js-mock is not anywhere close to complete, and it's not documented anywhere that the two projects are supposed to have the same version number.